blocklib/blocks/copy/copy.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <gnuradio/sync_block.hpp>
+namespace gr {
+namespace blocks {
+
+class copy : public sync_block
+{
+public:
+    typedef struct {
+        size_t itemsize;
+    } block_args;
+
+    enum class available_impl { CPU, CUDA };
```

Comment:
I've not experimented with CUDA and rfnoc and other hw-accelerated blocks. How will they be used? Does it make sense to have the implementation "hint" be passed in consistently (that is, not a per-block impl list, and containing specific device parameters, e.g. which GPU)?

Reply:
I haven't gotten to the point of abstracting the device specific properties that will be available per block, but am guessing there are a few ways to do this.  Don't have the answer yet, but will have some discussions and do some experimenting in this area.